presto-native-execution/scripts/dockerfiles/prestissimo-runtime.dockerfile

```diff
@@ -27,6 +27,7 @@ RUN mkdir -p /prestissimo /runtime-libraries
 COPY . /prestissimo/
 RUN EXTRA_CMAKE_FLAGS=${EXTRA_CMAKE_FLAGS} \
     make -j${NUM_THREADS} --directory="/prestissimo/" cmake-and-build BUILD_TYPE=${BUILD_TYPE} BUILD_DIR=${BUILD_DIR} BUILD_BASE_DIR=${BUILD_BASE_DIR}
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf.d/prestissimo.conf && echo "/usr/local/lib64" >> /etc/ld.so.conf.d/prestissimo.conf && ldconfig
 RUN ldd /prestissimo/${BUILD_BASE_DIR}/${BUILD_DIR}/presto_cpp/main/presto_server | awk 'NF == 4 { system("cp " $3 " /runtime-libraries") }'
```

Comment:
I feel we should ideally avoid ldconfig since a package installation usually should not depend on it.
If we copy the libraries to the corresponding lib or lib64, will that work?

Reply:
To clarify, we modify `| awk 'NF == 4 { system("cp " $3 " /runtime-libraries") }'` to install to lib and lib64.

Reply:
Ok, we can try to copy them to `/usr/lib` and see if that also fixes the problem.